RadixWallet/Features/SplashFeature/Splash.swift

```diff
@@ -82,8 +82,25 @@ public struct Splash: Sendable, FeatureReducer {
 	public func reduce(into state: inout State, viewAction: ViewAction) -> Effect<Action> {
 		switch viewAction {
 		case .appeared:
-			return .run { send in
-				let isAdvancedLockEnabled = await onboardingClient.loadProfile().appPreferences.security.isAdvancedLockEnabled
+			switch state.context {
+			case .appStarted:
+				return boot_sargon_os().concatenate(with: loadAdvancedLockState())
```

Comment:
Shouldn't the boot take place only on `.task`? In the end it is the same result since we ignore the `SargonOSAlreadyBooted` error, but would be cleaner to only boot it once and, if we ever get this error, at least log it.

Reply:
You mean to use task and not have the check of the `context`?

Reply:
yes

Reply:
Splash feature can be instantiated more than once, at app start and then on each foregrounding, this was added as part of Advanced Lock feature.